uw-frame-components/portal/widgets/partials/list-of-links.html

```diff
@@ -0,0 +1,34 @@
+<div class="widget-body list-of-links">
+
+  <!-- FOUR LINKS OR LESS -->
+  <div ng-show="config.links.length <= 4" class="list-of-links__{{config.links.length}}">
+    <circle-button ng-repeat="item in config.links"
+                   data-href="{{item.href}}"
+                   data-target="{{item.target}}"
+                   data-fa-icon="{{item.icon}}"
+                   data-disabled="false"
+                   data-title="{{item.title}}"
+                   data-trunclen="24">
+    </circle-button>
+  </div>
+
+  <!-- MORE THAN FOUR LINKS -->
+  <div ng-show="config.links.length > 4" class="list-of-links__long">
+    <ul class="widget-list">
+      <li ng-repeat='item in config.links | limitTo: 7' class="link-icon">
+        <i class="fa {{item.icon}}"></i>
+        <a ng-href="{{item.href}}" target="{{item.target}}">
+          {{item.title | truncate:24}}
+        </a>
+      </li>
+    </ul>
+  </div>
+
+</div>
+
+<!-- LAUNCH BUTTON -->
+<launch-button ng-if='config.launchText && widget.url'
+               data-href="{{widget.url}}"
+               data-target="{{widget.target}}"
```

Comment:
Could this be more secure if it included [`rel="noopener"`](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/)?